### DISCUSSION TITLE
migrating from clr-icon to cds-icon

### DISCUSSION THREAD

**Discussion by kikemarquez8** [vmware-clarity/core#171] [Q&A] [answered]
Hey guys, I have a small question regarding the migration from clr-icons to core. 
I read this article about migrating from cds to clr icons [article](https://medium.com/claritydesignsystem/goodbye-clarity-icons-hello-clarity-core-adb07de9739e), but if I go to the [clarity website](https://clarity.design/icons/get-started) on how to use icons it still shows clr-icons, is this intentional?

**Comment by ashleyryan** [comment, marked as answer]
It's not intentional. Our websites are in a rough place right now and I'm working on making sure some of the older documentation is switched over to use cds-icon. I'll see about getting that updated

**Reply by kikemarquez8** [reply]
Thanks Ashley, no worries just wanted to make sure
